src/Nri/Ui/AssignmentIcon/V1.elm

```diff
@@ -277,3 +278,232 @@ startSecondary =
+                    , Svg.path [ Attributes.d "M12.35615,31.837588 C11.57915,31.837588 10.9461857,31.2122359 10.9461857,30.4435739 C10.9461857,29.6788204 11.5765071,29.0534683 12.3521857,29.0534683 L20.8740786,29.0534683 C21.9140429,29.0534683 22.2840429,29.8038908 22.2840429,30.4461796 C22.2840429,31.0884683 21.9140429,31.837588 20.8740786,31.837588 L12.35615,31.837588 Z" ] []
+                    , Svg.path [ Attributes.d "M12.35615,23.2128099 C11.5804714,23.2128099 10.9475071,22.5887606 10.9461857,21.8187958 C10.9461857,21.0540423 11.5765071,20.429993 12.3521857,20.429993 L35.5155071,20.429993 C36.5554714,20.429993 36.9254714,21.1791127 36.9254714,21.8214014 C36.9254714,22.4636901 36.5554714,23.2128099 35.5155071,23.2128099 L12.35615,23.2128099 Z" ] []
+                    , Svg.path [ Attributes.d "M12.35615,14.2444789 C11.5804714,14.2444789 10.9475071,13.6204296 10.9461857,12.8504648 C10.9461857,12.0857113 11.5765071,11.461662 12.3521857,11.461662 L35.5155071,11.461662 C36.5554714,11.461662 36.9254714,12.2107817 36.9254714,12.8530704 C36.9254714,13.4953592 36.5554714,14.2444789 35.5155071,14.2444789 L12.35615,14.2444789 Z" ] []
+                    , Svg.path [ Attributes.d "M12.35615,5.62347887 C11.6227571,5.62347887 11.0188643,5.06717606 10.9527929,4.35974648 L10.9527929,4.23207042 C10.9527929,3.46471127 11.5831143,2.83935915 12.3587929,2.83935915 L35.5155071,2.83935915 C36.5554714,2.83935915 36.9254714,3.58978169 36.9254714,4.23207042 C36.9254714,4.87435915 36.5554714,5.62347887 35.5155071,5.62347887 L12.35615,5.62347887 Z" ] []
+                    ]
+                ]
+            ]
+        ]
+        |> Nri.Ui.Svg.V1.fromHtml
+
+
+{-| -}
+assignArrow : Nri.Ui.Svg.V1.Svg
```

Comment:
🤔 I wonder if this arrow should go in `UiIcon` in the arrows section. Do you think it makes more sense for it to be here..?